<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Git View</string>
  </property>
  <property name="windowIcon">
   <iconset theme="Git" resource="resource.qrc">
    <normaloff>:/resource/logo@2x.png</normaloff>:/resource/logo@2x.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="iconSize">
   <size>
    <width>98</width>
    <height>92</height>
   </size>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>905</width>
     <height>455</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="topLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutTool">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutForTreeView">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutForTreeViewHead">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="comboShowItems">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>33</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Show files depending on git status</string>
            </property>
            <item>
             <property name="text">
              <string>Show all files</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Show all git actions</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Show modified</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Show added</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Show deleted</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Show unknown</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Show staged</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Show unmerged</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckHideEmptyParent">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Hides parent tree item, if it does not contain entries</string>
            </property>
            <property name="text">
             <string>Hide empty Parent</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckShortState">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Show short git status</string>
            </property>
            <property name="text">
             <string>Short State</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeSource">
          <property name="toolTip">
           <string>Show file and folder tree of git folder</string>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>4</number>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>State</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutForTextBrowser">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutForTextBrowserHead">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelHeader">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>File/Text:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelFilePath">
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCloseText">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Close file or text view</string>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStoreText">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Save changed text to file</string>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutHistoryAndText">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QHistoryTreeWidget" name="treeHistory">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Shows git history of files or folders</string>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectItems</enum>
              </property>
              <column>
               <property name="text">
                <string>History</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Author</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QBranchTreeWidget" name="treeBranches">
              <property name="toolTip">
               <string>Shows git branches</string>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectItems</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <column>
               <property name="text">
                <string>Branch</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="toolTip">
             <string>Shows git command results or
File content opened with double click</string>
            </property>
            <property name="undoRedoEnabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutFlags">
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>File Flags:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSystemFiles">
        <property name="toolTip">
         <string>Show system files in git folder</string>
        </property>
        <property name="text">
         <string>System Files</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckHiddenFiles">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Show hidden files in git folder</string>
        </property>
        <property name="text">
         <string>Hidden Files</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSymbolicLinks">
        <property name="toolTip">
         <string>Show symbolic links in git folder</string>
        </property>
        <property name="text">
         <string>Symbolic Links</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckDirectories">
        <property name="toolTip">
         <string>Show folders in git folder</string>
        </property>
        <property name="text">
         <string>Directories</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckFiles">
        <property name="toolTip">
         <string>Show files in git folder</string>
        </property>
        <property name="text">
         <string>Files</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSelect_source_folder">
   <property name="text">
    <string>Select source folder...</string>
   </property>
  </action>
  <action name="actionSelect_Destination">
   <property name="text">
    <string>Select Destination...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QBranchTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">qbranchtreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QHistoryTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">qhistorytreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
